README.md

```diff
@@ -42,16 +42,20 @@ We are using this on a variety of internal projects within the Conservation
 Biology Institute.
 
 
-## Installation
-[pipenv](https://github.com/pypa/pipenv) is used for managing dependencies in this project.
-
-`pipenv install trefoil`
+## Development
+[Poetry](https://python-poetry.org/docs/) is used for managing dependencies in this project.
+To develop this project, install its dependencies with:
 
+``` text
+$ poetry install
+```
 
-### No longer directly maintained / supported:
-On Windows, install the ones that require compiling from [Python Windows Packages](http://www.lfd.uci.edu/~gohlke/pythonlibs/).
-Then install the remainder using `pip`
+## Installation
+Trefoil is available on PyPi. To install, use:
 
+```text
+$ pip install trefoil
+```
```

Comment:
Let's move "Installation" above the "Development" section since it's the more common thing people would want to do.